xcvm/cosmwasm/contracts/asset-registry/examples/schema.rs

```diff
@@ -0,0 +1,17 @@
+use std::env::current_dir;
+use std::fs::create_dir_all;
+
+use cosmwasm_schema::{export_schema, remove_schemas, schema_for};
+
+use xcvm_asset_registry::msg::{ExecuteMsg, InstantiateMsg, QueryMsg};
+
+fn main() {
+    let mut out_dir = current_dir().unwrap();
+    out_dir.push("schema");
+    create_dir_all(&out_dir).unwrap();
+    remove_schemas(&out_dir).unwrap();
+
+    export_schema(&schema_for!(InstantiateMsg), &out_dir);
+    export_schema(&schema_for!(ExecuteMsg), &out_dir);
+    export_schema(&schema_for!(QueryMsg), &out_dir);
+}
```

Comment:
Why is this in a `examples` directory?

Reply:
So that it can easily be run to create schemas. I would prefer this to be in scripts directory alongside with a script to build and run. But I didn't want to change this since all of the contracts that are provided and the template use this convention. We can have a script to generate schema for all or specified contracts if you want.

Reply:
This is following CosmWasm way of easily exporting schemas